models/_sector/nft/trades/chains/solana/platforms/_schema.yml

```diff
@@ -159,6 +159,83 @@ models:
        tests:
          - not_null
 
+  - name: magiceden_v3_solana_trades
+    meta:
+      blockchain: solana
+      project: tensor
+      contributors: [ tsekityam ]
+    config:
+      tags: [ 'magiceden','solana','nft' ]
+    description: >
+      magiceden solana trades
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - project
+            - trade_category
+            - outer_instruction_index
+            - inner_instruction_index
+            - account_merkle_tree
+            - leaf_id
+            - tx_id
+      - check_seed:
+          seed_file: ref('magiceden_v3_solana_trades_seed')
+          match_columns:
+            - block_slot
+            - tx_id
+            - outer_instruction_index
+            - inner_instruction_index
```

Comment:
ideally the unique key columns = the match_columns when joining to seed to perform tests. is it possible the unique key contains more columns than necessary?

Reply:
at this time, `nft_solana.trades` is a standalone spell -- as in, doesn't flow into `nft.trades`

i think it's okay to continue on that path until ready to merge the two. i know @andrewhong5297 has been owning this path of solana spells for now, so we can continue with his approval

in the future, it'll be good to have you work with @0xRobin on how to merge into `nft.trades`

anyway, that's all a side note for now. just reminding everyone :)

Reply:
it's nice to design the dbt lineage where all models feeding into a union'd spell downstream contain the same set of unique keys, similar to how `nft.trades` has been redesigned. i'm sure it'll become more complex when we merge evm models with solana

Reply:
@jeff-dude , I reviewed the matched columns and unique key columns for the seed file, PTAL.